src/code/z_camera.c

```diff
@@ -1,22 +1,98 @@
+#include "prevent_bss_reordering.h"
 #include "global.h"
 #include "z64quake.h"
 #include "z64shrink_window.h"
 #include "z64view.h"
+#include "overlays/actors/ovl_En_Horse/z_en_horse.h"
```

Comment:
Could you move these below the `@file description`? That is what I see for all other files with it currently.

Reply:
Bump